Bookish/Views/Home/Index.cshtml

```diff
@@ -1,24 +1,37 @@
 ﻿@{
     ViewData["Title"] = "Home Page";
 }
-@using Bookish.Models
-@model BookInputModel
 
+@model DefaultInputModel
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
     @using (Html.BeginForm("BookEntry", "Catalog", FormMethod.Get))
     {
+        
         <fieldset>
-            
             <legend>Enter a book:</legend>
-            @Html.TextBoxFor(m => m.BookName, new{@type="text", placeholder="Book name"})
-            @Html.TextBoxFor(m => m.ISBN, new{@type="text", placeholder="ISBN", pattern="[0-9]{13}", title = "Please enter a 13 digit number"})
-            @Html.TextBoxFor(m => m.Publisher, new{@type="text", placeholder="Publisher"})
-            @Html.TextBoxFor(m => m.DatePublished, new{@type="date", placeholder="Publish date"})
-            @Html.TextBoxFor(m => m.Author, new{@type="text", placeholder="Author"})
+            @Html.TextBoxFor(m => m.BookInput.BookName, new{@type="text", placeholder="Book name"})
+            @Html.TextBoxFor(m => m.BookInput.ISBN, new{@type="text", placeholder="ISBN", pattern="[0-9]{13}", title = "Please enter a 13 digit number"})

```

Comment:
If you want to be strict, you could find the _real_ pattern for ISBN numbers

Reply:
The real pattern has random hyphens in it, I will deal with this in a future branch